### PR TITLE
[WasmGC article] Add Flutter aside

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add Flutter aside

### CODE REVIEW

File: site/en/blog/wasmgc/index.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Now you may be wondering what the point is, since the Kotlin code above basicall
 
 {% Glitch id="kotlin-wasm-image-viewer", height="900" %}
 
-{% Aside 'warning' %}
+{% Aside %}
```
Comment (rachelandrew) [inline site/en/blog/wasmgc/index.md:123]: Why are we removing the accessibility information? Without that, we aren't explaining why removing these features is a problem.

Comment (tomayac) [inline site/en/blog/wasmgc/index.md:123]:
```suggestion
{% Aside 'warning' %}
```

File: site/en/blog/wasmgc/index.md [hunk 2]
```diff
@@ -130,6 +130,10 @@ The Dart and Flutter teams at Google are also preparing support for WasmGC. The
 
 <iframe style="width: 100%; height: 800px" src="https://flutterweb-wasm.web.app/"></iframe>
 
+{% Aside %}
```
Comment (tomayac) [inline site/en/blog/wasmgc/index.md:133]:
```suggestion
{% Aside 'warning' %}
```

File: site/en/blog/wasmgc/index.md (hunk 1 above)
Comment (tomayac) [reply site/en/blog/wasmgc/index.md]: Let's add it to both then.

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**